packages/gestalt/src/Modal.js

```diff
@@ -10,9 +10,12 @@ import Heading from './Heading.js';
 import StopScrollBehavior from './behaviors/StopScrollBehavior.js';
 import Text from './Text.js';
 import TrapFocusBehavior from './behaviors/TrapFocusBehavior.js';
+import { ScrollableContainerWithForwardRef as InternalScrollableContainer } from './ScrollableContainer.js';
+import { ScrollableContainerProvider } from './contexts/ScrollableContainer.js';
 import modalStyles from './Modal.css';
 
 type Props = {|
+  _dangerouslySetExperimentalProp?: boolean, // Temporary undocumented prop to support experimentation inside Modal and Sheet.
```

Comment:
I think this name could be more specific…something like `_dangerousScrollableExperimentEnabled`?

Reply:
DONE